src/hardware/openbench-logic-sniffer/protocol.c

```diff
@@ -70,46 +77,39 @@ SR_PRIV int ols_send_reset(struct sr_serial_dev_inst *serial)
 }
 
 /* Configures the channel mask based on which channels are enabled. */
-SR_PRIV void ols_channel_mask(const struct sr_dev_inst *sdi)
+SR_PRIV uint32_t ols_channel_mask(const struct sr_dev_inst *sdi)
 {
-	struct dev_context *devc;
-	struct sr_channel *channel;
-	const GSList *l;
-
-	devc = sdi->priv;
-
-	devc->channel_mask = 0;
-	for (l = sdi->channels; l; l = l->next) {
-		channel = l->data;
+	uint32_t channel_mask = 0;
+	for (const GSList *l = sdi->channels; l; l = l->next) {
+		struct sr_channel *channel = l->data;
 		if (channel->enabled)
-			devc->channel_mask |= 1 << channel->index;
+			channel_mask |= 1 << channel->index;
 	}
+
+	return channel_mask;
 }
 
-SR_PRIV int ols_convert_trigger(const struct sr_dev_inst *sdi)
+static int convert_trigger(const struct sr_dev_inst *sdi, struct ols_basic_trigger_desc *ols_trigger)
```

Comment:
Why no 'ols_' prefix in the function name? I think the namespace is a bit generic this way.

Reply:
I thought since they are private to the compilation unit, I could omit the prefix, making the names shorter. Hmm, but that's not how the file is authored, I see. I'll add some more `ols_` prefixes…